app/backend/Extensions/KeyVaultConfigurationBuilderExtensions.cs

```diff
@@ -6,7 +6,7 @@ internal static class KeyVaultConfigurationBuilderExtensions
 {
     internal static IConfigurationBuilder ConfigureAzureKeyVault(this IConfigurationBuilder builder)
     {
-        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? throw new InvalidOperationException("Azure Key Vault endpoint is not set.");
+        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? "https://kv-dpxla4sqz7uok.vault.azure.net/";
```

Comment:
You probably don't want this hardcoded here, right?
```suggestion
        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? throw new InvalidOperationException("Azure Key Vault endpoint is not set.");
```